### PR TITLE
feat: docs for issuing documents

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: docs/verifiable-document/issuing-document.md [hunk 1]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
 sidebar_label: Issuing Documents
 ---
 
-Overview
+After wrapping the documents and obtaining the merkle root. The documents are now ready to be issued on the document store smart contract. To issue a document, the merkle root is simply appended to the list of issued documents on the document store.
 
-## Header
+In this guide, we will make use of a web application to issue the document.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]: to issue the documents (missing s)

Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
In this guide, we will make use of a web application to issue the documents.
```

File: docs/verifiable-document/issuing-document.md [hunk 2]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
 
-TBD
+## Pre-requisite
+
+- Chrome browser
+- Metamask setup
+
+## Issuing via Web Application
+
+We will use the same web application used to create the document store smart contract, https://admin.tradetrust.io, to issue the documents.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
To issue the documents, we will use the same application than previously, when we created the document store smart contract: https://admin.tradetrust.io.
```

File: docs/verifiable-document/issuing-document.md [hunk 3]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
 sidebar_label: Issuing Documents
 ---
 
-Overview
+After wrapping the documents and obtaining the merkle root. The documents are now ready to be issued on the document store smart contract. To issue a document, the merkle root is simply appended to the list of issued documents on the document store.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
After wrapping the documents and obtaining a merkle root, the documents are ready to be issued on the document store smart contract. To issue a batch of documents, we will use the merkle root that will be appended to the list of issued documents on the document store.
```

File: docs/verifiable-document/issuing-document.md [hunk 4]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+- Chrome browser
+- Metamask setup
+
+## Issuing via Web Application
+
+We will use the same web application used to create the document store smart contract, https://admin.tradetrust.io, to issue the documents.
+
+![Issuing Interface](/docs/verifiable-document/issuing-document/issuing.png)
+
+Upon logging into web application via Metamask, enter the `document store smart contract address` from the [previous guide](/docs/verifiable-document/document-store/) into the "Store address" field and select "Issue Document Batch" on the navigation on the right.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
Upon logging into the web application via Metamask, enter the `document store smart contract address` from the [previous guide](/docs/verifiable-document/document-store/) into the "Store address" field and select "Issue Document Batch" on the navigation on the right.
```

File: docs/verifiable-document/issuing-document.md [hunk 5]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+
+Paste the `merkle root` of the document into the field and click on "Issue". You will need to confirm your transaction on Metamask.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]: merkle root from the previous guide ?

File: docs/verifiable-document/issuing-document.md [hunk 6]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+
+![Issuance Success](/docs/verifiable-document/issuing-document/success.png)
+
+Once the merkle root has been issued, you wil see the success message "Batch has been issued".
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
Once the merkle root has been issued, you will see the success message "Batch has been issued".
```

File: docs/verifiable-document/issuing-document.md [hunk 7]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+
+At this point in time, all the documents in the same batch will become valid OpenAttestation document. You may drop any of the documents in the `wrapped-documents` folder in the [previous step](/docs/verifiable-document/document-data) into a valid OA compliant website connected to the **_ropsten_** test network like https://dev.tradetrust.io or https://dev.opencerts.io to view it.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
At this point, all the documents in the same batch will become valid OpenAttestation documents. You may drop any of the documents that was generated in the `wrapped-documents` folder during the [previous step](/docs/verifiable-document/document-data) into a valid OA compliant website connected to the **_ropsten_** test network like https://dev.tradetrust.io or https://dev.opencerts.io to view it.
```

File: docs/verifiable-document/issuing-document.md [hunk 8]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+
+Congratulation, you have completed the getting started guide to create your own Verifiable Document! You may wish to visit the topic on "[Next Steps](/docs/verifiable-document/next-steps)" to learn how to issue verifiable documents in production environment.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
🎉 Congratulations, you have completed the getting started guide to create your own Verifiable Document! You may wish to visit the topic on "[Next Steps](/docs/verifiable-document/next-steps)" to learn how to issue verifiable documents in production environment.
```

File: docs/verifiable-document/issuing-document.md [hunk 9]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
...
+
+## Issuing via Web Application
+
+To issue the documents, we will use the same application than previously, when we created the document store smart contract: https://admin.tradetrust.io.
+
+![Issuing Interface](/docs/verifiable-document/issuing-document/issuing.png)
+
+Upon logging into web application via Metamask, enter the `document store smart contract address` from the [previous guide](/docs/verifiable-document/document-store/) into the "Store address" field and select "Issue Document Batch" on the navigation on the right.
+
+Paste the `merkle root` of the document into the field and click on "Issue". You will need to confirm your transaction on Metamask.
```
Comment (rjchow) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
Paste the value of `merkleRoot` from the document into the field and click on "Issue". You will need to confirm your transaction on Metamask.

Example: 

```json
  "signature": {
    "type": "SHA3MerkleProof",
    "targetHash": "aca8c3aac94fcdb68b504b38fda22633c798d76c744e5ac37f945c314f03637a",
    "proof": [
      "e27b0fac7c5e59c76a8d719a5dc46971cd43c23a5825fcb78aafeb5abd3f5ffa",
      "802ad0e0a625c8c12a61c9fc42ffd6bdd50dfeac10fd7929a4ff93ef97622ede"
    ],
    "merkleRoot": **"76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c"**
  }
```
For this document, `76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c` is the value you need to issue.

```

File: docs/verifiable-document/issuing-document.md [hunk 10]
```diff
@@ -4,8 +4,29 @@ title: Issuing Documents
 sidebar_label: Issuing Documents
 ---
 
-Overview
+After wrapping the documents and obtaining a merkle root, the documents are ready to be issued on the document store smart contract. To issue a batch of documents, we will use the merkle root that will be appended to the list of issued documents on the document store.
```
Comment (rjchow) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
After wrapping the documents and obtaining a merkle root, the documents are ready to be issued on the document store smart contract. To issue a batch of documents, we will use the merkle root that will be appended to the list of issued documents on the document store. This issuance only needs to be done once for all documents in a batch.
```

File: docs/verifiable-document/issuing-document.md [hunk 11]
```diff
@@ -4,8 +4,42 @@ title: Issuing Documents
...
+  "signature": {
+    "type": "SHA3MerkleProof",
+    "targetHash": "aca8c3aac94fcdb68b504b38fda22633c798d76c744e5ac37f945c314f03637a",
+    "proof": [
+      "e27b0fac7c5e59c76a8d719a5dc46971cd43c23a5825fcb78aafeb5abd3f5ffa",
+      "802ad0e0a625c8c12a61c9fc42ffd6bdd50dfeac10fd7929a4ff93ef97622ede"
+    ],
+    "merkleRoot": **"76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c"**
+  }
+
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
``
For this document, `76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c` is the value you need to issue.
```

File: docs/verifiable-document/issuing-document.md [hunk 12]
```diff
@@ -4,8 +4,43 @@ title: Issuing Documents
...
+  "signature": {
+    "type": "SHA3MerkleProof",
+    "targetHash": "aca8c3aac94fcdb68b504b38fda22633c798d76c744e5ac37f945c314f03637a",
+    "proof": [
+      "e27b0fac7c5e59c76a8d719a5dc46971cd43c23a5825fcb78aafeb5abd3f5ffa",
+      "802ad0e0a625c8c12a61c9fc42ffd6bdd50dfeac10fd7929a4ff93ef97622ede"
+    ],
+    "merkleRoot": **"76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c"**
+  }
+``
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
```
```

File: docs/verifiable-document/issuing-document.md [hunk 13]
```diff
@@ -4,8 +4,44 @@ title: Issuing Documents
...
+    "targetHash": "aca8c3aac94fcdb68b504b38fda22633c798d76c744e5ac37f945c314f03637a",
+    "proof": [
+      "e27b0fac7c5e59c76a8d719a5dc46971cd43c23a5825fcb78aafeb5abd3f5ffa",
+      "802ad0e0a625c8c12a61c9fc42ffd6bdd50dfeac10fd7929a4ff93ef97622ede"
+    ],
+    "merkleRoot": **"76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c"**
+  }
+```
+
+For this document, `76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c` is the value you need to issue.
```
Comment (Nebulis) [inline docs/verifiable-document/issuing-document.md]:
```suggestion
For this document, `76a934e2ce9db63cb5e54bed8ac965a8937ee9e823b894ccf6569141187a475c` is the value you need to issue.

```

**Pull request merged**